task/01-strings-tasks.js

```diff
@@ -21,9 +19,10 @@
  *   'aa',''    => 'aa'
  *   '',  'bb'  => 'bb'
  */
-function concatenateStrings(value1, value2) {
```

Comment:
I reckon, we can leave normal functions here.

Reply:
There is a question whether it is better to stay with functional declarations or move to functional expressions
I believe that functional expression is more appropriate approach.

Reply:
Taking into account the educational nature of the app, I agree that functional declaration is better.